lib/shared/utils.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2017 resin.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const _ = require('lodash');
+const flatten = require('flat').flatten;
+
+/**
+ * @summary Create a flattened copy of the object with all keys transformed in start case
+ *
+ * @param {Object} object - object to transform
+ * @return {Object}
```

Comment:
Good idea. See https://github.com/resin-io/etcher/pull/1147. I didn't enable the rule to require a return description though, since it requires them also for functions that return promises that resolve to `undefined`, which looks weird. I'll see if I can find something else for that.